src/main/scala/firrtl/stage/FirrtlOptions.scala

```diff
@@ -0,0 +1,93 @@
+// See LICENSE for license details.
+
+package firrtl.stage
+
+import scala.util.Try
+
+import firrtl._
+import firrtl.options.{OptionsException, StageOptions}
+import firrtl.annotations.Annotation
+import firrtl.ir.Circuit
+import logger.LogLevel
+
+/** Internal options used to control the FIRRTL compiler stage
+  *
+  * @param topName the name of the top module
```

Comment:
First five tags here appear to be fossils

Reply:
Thanks. I removed the fossil tags and added the missing tags.